esmvalcore/cmor/check.py

```diff
@@ -178,9 +178,35 @@ def _check_var_metadata(self):
         # Check standard_name
         if self._cmor_var.standard_name:
             if self._cube.standard_name != self._cmor_var.standard_name:
-                self.report_error(
-                    self._attr_msg, self._cube.var_name, 'standard_name',
-                    self._cmor_var.standard_name, self._cube.standard_name)
+                if self.automatic_fixes:
+                    self.report_warning(
+                        'Standard name for %s changed from %s to %s',
+                        self._cube.var_name,
+                        self._cube.standard_name,
+                        self._cmor_var.standard_name
+                    )
+                    self._cube.standard_name = self._cmor_var.standard_name
+                else:
+                    self.report_error(
+                        self._attr_msg, self._cube.var_name, 'standard_name',
+                        self._cmor_var.standard_name, self._cube.standard_name
+                    )
+        # Check long_name
+        if self._cmor_var.long_name:
+            if self._cube.long_name != self._cmor_var.long_name:
+                if self.automatic_fixes:
+                    self.report_warning(
+                        'Long name for %s changed from %s to %s',
+                        self._cube.var_name,
+                        self._cube.long_name,
+                        self._cmor_var.long_name
+                    )
+                    self._cube.long_name = self._cmor_var.long_name
+                else:
+                    self.report_warning(
```

Comment:
```suggestion
                    self.report_error(
```
?

Reply:
Long name are not that used for the purpose of getting the data. People usually rely on standard and var names. I just did not want to add an extra source of errors (specifically because we were not testing it before)

Reply:
But we're fixing it automatically now, right? So it will never cause an error, unless you disable `fix_metadata` in the recipe.